### PR TITLE
Configure Travis CI

### PR DESCRIPTION
**Pull request by yugui**
* Upgrade protoc version
* Upgrade go version
* Make CI faster and strict

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,26 +1,29 @@
+branches:
+  only:
+    - master
 language: go
-
+sudo: false
 go:
-- 1.3
+- 1.4
```
Comment (shurcooL) [inline .travis.yml]: Why 1.4 and not 1.5, which is the latest official release?

Comment (yugui) [inline .travis.yml]: certainly. fixed.

**Comment by hayate** [comment]
LGTM

File: .travis.yml (hunk 1 above)
Comment (shurcooL) [inline .travis.yml]: Thanks!

**Pull request merged**